tools/postman/scripts/fetch.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+#########################################################
+# Fetch openapi from remote file
+# Environment variables:
+#   OPENAPI_FILE_NAME - openapi file name to use
+#   OPENAPI_FOLDER - folder for saving openapi file
+#   API_BASE_URL - base URL where the spec is hosted
+#   S3_BUCKET - S3 bucket where the spec is hosted
+#   VERSIONS_FILE - openapi versions file name to use
+#########################################################
+
+OPENAPI_FILE_NAME=${OPENAPI_FILE_NAME:-"atlas-api.json"}
+OPENAPI_FOLDER=${OPENAPI_FOLDER:-"../openapi"}
+API_BASE_URL=${API_BASE_URL:-"https://cloud.mongodb.com"}
+S3_BUCKET=${S3_BUCKET:-"mongodb-mms-prod-build-server"}
+VERSIONS_FILE=${VERSIONS_FILE:-"versions.json"}
+
+versions_url="${API_BASE_URL}/api/openapi/versions"
+
+pushd "${OPENAPI_FOLDER}"
+
+echo "Fetching versions from $versions_url"
+curl --show-error --fail --silent -o "${VERSIONS_FILE}" \
+     -H "Accept: application/json" "${versions_url}"
+
+## Dynamic Versioned API Version
+CURRENT_API_REVISION=$(jq -r '.versions."2.0" | .[-1]' < "./${VERSIONS_FILE}")
+
+echo "Fetching OpenAPI release sha"
+sha=$(curl --show-error --fail --silent -H "Accept: text/plain" "${API_BASE_URL}/api/private/unauth/version")
+
+echo "Fetching OAS file for ${sha}"
+openapi_url="https://${S3_BUCKET}.s3.amazonaws.com/openapi/${sha}-v2-${CURRENT_API_REVISION}.json"
```

Comment:
@matt-condon This code is only temporary measure until we get the actual openapi files directly in the repository root.

Reply:
> will the OAS we retrieve always conform to this pattern

It will change once we move to the mongodb/openapi but we will update this as part of https://jira.mongodb.org/browse/CLOUDP-261859

Reply:
Summary from chat with the team:
- We do not need to keep openapi file for the diff (we will always push the changes in the job)
- Current Github actions will be only triggered manually

Once we have done jira that @andreaangiolillo shared we can trigger the job manually when the openapi changes are detected.